build-support/bin/ci.py

```diff
@@ -338,12 +344,16 @@ def run_python_tests_v2() -> None:
         "tests/python::",
       ], stdout=subprocess.PIPE, encoding="utf-8", check=True).stdout.strip().split("\n")
       v2_targets = set(all_targets) - blacklisted_targets
-      subprocess.run([
-        "./pants.pex",
-        "--no-v1",
-        "--v2",
-        "test.pytest"
-      ] + sorted(v2_targets) + PYTEST_PASSTHRU_ARGS, check=True)
+      v2_flags = ["--no-v1", "--v2"]
+      if remote_execution_oauth_token_path is not None:
+        v2_flags.extend([
+          "--pants-config-files=['pants.remote.ini', 'pants.travis-ci.ini']",
```

Comment:
AFAIK, this syntax will cause `pants.ini` to not be used at all. If you want it used, would need to include it in this list. See https://www.pantsbuild.org/options.html#list-options

Reply:
Oh neat, didn't realize there were so many ways to do this. Will revisit and find the simplest possible.